CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+
+## [v1.4.0](https://github.com/provenance-io/provenance/releases/tag/v1.4.0) - 2021-06-02
```

Comment:
is this in prep for the release, a little confused about the unreleased section

Reply:
yeah ... we always keep that header at the top... it makes more sense when the first change shows up on main though